### PR TITLE
Fixed multiple typos in Action Manager

### PR DESCRIPTION
**Pull request by dtoncu**
This commit fixes typos regarding code and docstrings.

### CODE REVIEW

File: argus/action_manager/windows.py [hunk 1]
```diff
@@ -313,15 +313,16 @@ def mkfile(self, path):
...
+                        " LastAccessTime will be updated.".format(path))
             self._client.run_command_with_retry(
                 "echo $null >> '{}'".format(path),
                 command_type=util.POWERSHELL)
         elif self.is_dir(path):
             raise exceptions.ArgusCLIError(
                 "Path '{}' leads to a"
                 " directory.".format(path))
-        self._new_item(path, self._FILE)
+        else:
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py]: This isn't a code typo, you could change the commit message to be more explicit.

Comment (dtoncu) [reply argus/action_manager/windows.py]: Done.

Comment (ader1990) [reply argus/action_manager/windows.py]: remove this change and good to merge

**Review by ader1990 — CHANGES_REQUESTED** [review]

File: argus/action_manager/windows.py [hunk 2]
```diff
@@ -313,7 +313,7 @@ def mkfile(self, path):
         """
         if self.is_file(path):
             LOG.warning("File '{}' already exists. LastWriteTime and"
-                        " LastAccessTime will be updated.").format(path)
```
Comment (ader1990) [inline argus/action_manager/windows.py:316]: remove this part, also

**Review by ader1990 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**